### ISSUE TITLE
[Bug] AllowedInclude::relationship and AllowedInclude::count can't be used

### ISSUE THREAD

**Issue by Sharptsa** [spatie/laravel-query-builder#412]
It is currently impossible to use in `allowedIncludes()` : 

- `AllowedInclude::relationship('name', 'internalName')` for aliasing relationships 
- `AllowedInclude::count('nameCount', 'internalNameCount')` as [stated in the docs](https://docs.spatie.be/laravel-query-builder/v2/features/including-relationships/#including-related-model-count)

While aliasing sort with `AllowedSort::field('fieldAlias', 'field')` works fine.

Example:
```php
$models = QueryBuilder::for(Model::class)
    ->allowedIncludes([
        AllowedInclude::count('relationshipCount'),
        AllowedInclude::relationship('relationshipAlias', 'relationship')
    ])
    ->get();
```
Trying to use those includes will result in the following error:
```json
{
    "message": "Requested include(s) `relationshipCount, relationshipAlias` are not allowed. Allowed include(s) are `[{}], [{}]Count, [{},{}], [{},{}]Count`.",
    "statusCode": 400
}
```

I believe the issue is located here:

https://github.com/spatie/laravel-query-builder/blob/d0d82095138f961241c81efe0a40147009ad6d4d/src/Concerns/AddsIncludesToQuery.php#L22-L24

Because `AllowedInclude::relationship()` and `AllowedInclude::count('nameCount', 'internalNameCount')` already return a Collection, it never passes in this condition

**Comment by dominikb** [comment]
Here's a failing test to reproduce this issue:

```php
/** @test */
public function it_can_alias_multiple_allowed_includes()
{
    $request = new Request([
        'include' => 'relatedModelsCount,relationShipAlias'
    ]);

    $models = QueryBuilder::for(TestModel::class, $request)
        ->allowedIncludes([
            AllowedInclude::count('relatedModelsCount'),
            AllowedInclude::relationship('relationShipAlias', 'otherRelatedModels')
        ]);

    dd($models->first());
}
```

```
Spatie\QueryBuilder\Exceptions\InvalidIncludeQuery : Requested include(s) `relatedModelsCount, relationShipAlias` are not allowed. Allowed include(s) are `[{}], [{}]Count, [{},{}], [{},{}]Count`.
 /Users/dominik/code/laravel-query-builder/src/Exceptions/InvalidIncludeQuery.php:30
 /Users/dominik/code/laravel-query-builder/src/Concerns/AddsIncludesToQuery.php:71
 /Users/dominik/code/laravel-query-builder/src/Concerns/AddsIncludesToQuery.php:36
 /Users/dominik/code/laravel-query-builder/tests/IncludeTest.php:293
```

I'm looking into this.

**Comment by dominikb** [comment]
@Sharptsa I've just opened a PR that should fix this issue, at least it fixes my test.
Can you try this change in your codebase and provide feedback?

**Comment by AlexVanderbist** [comment]
Fix tagged in 2.6.1 thanks to @dominikb !
